include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -166,6 +167,7 @@ class argument_parser
      * \param[in] argc           The number of command line arguments.
      * \param[in] argv           The command line arguments to parse.
      * \param[in] version_check  Notify users about app version updates (default true).
+     * \param[in] subcommands    A list of sub-programs (see \link subcommand_arg_parse subcommand parsing \endlink).
```

Comment:
Why do you use the term "sub-program" here when you use subcommand everywhere else?